src/SurveyService.Mappers/Db/DbOptionMapper.cs

```diff
@@ -0,0 +1,37 @@
+﻿using LT.DigitalOffice.Kernel.Extensions;
+using LT.DigitalOffice.SurveyService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.SurveyService.Models.Db;
+using LT.DigitalOffice.SurveyService.Models.Dto;
+using Microsoft.AspNetCore.Http;
+using System;
+
+namespace LT.DigitalOffice.SurveyService.Mappers.Db;
+
+public class DbOptionMapper : IDbOptionMapper
+{
+  private readonly IHttpContextAccessor _httpContextAccessor;
+
+  public DbOptionMapper(
+    IHttpContextAccessor httpContextAccessor)
+  {
+    _httpContextAccessor = httpContextAccessor;
+  }
+
+  public DbOption Map(CreateOptionRequest request)
+  {
+    return request is null
+      ? null
+      : new DbOption
+      {
+        Id = Guid.NewGuid(),
+        QuestionId = request.QuestionId,
+        Content = request.Content,
+        IsCustom = request.IsCustom,
+        IsActive = true,
+        CreatedBy = _httpContextAccessor.HttpContext.GetUserId(),
+        CreatedAtUtc = DateTime.UtcNow,
+        ModifiedBy = _httpContextAccessor.HttpContext.GetUserId(),
+        ModifiedAtUtc = DateTime.UtcNow
+      };
```

Comment:
don't need ModifiedBy and ModifiedAtUtc in this mapper at all

Reply:
Done